tests/cli/test_datasets.py

```diff
@@ -622,3 +622,49 @@ def test_dataset_rm(tmpdir, runner, project, client):
 
     result = runner.invoke(cli.cli, ['doctor'], catch_exceptions=False)
     assert 0 == result.exit_code
+
+
+@pytest.mark.integration
+def test_dataset_import_real_doi(runner, project):
+    """Test dataset import for existing DOI."""
+    result = runner.invoke(
+        cli.cli, ['dataset', 'import', '10.5281/zenodo.597964'], input='y'
```

Comment:
is it possible to mock these?

Reply:
Since their API can change I wouldn't mock. I marked them as integration and now on the runner, we need to split the unit against integration tests. We have a couple of integrations already which are not marked/separated.

Reply:
oh I see, I didn't realize that was an option :)